src/index.ts

```diff
@@ -10,18 +10,37 @@
  */
 
 import * as ts from 'typescript';
+let argv = require('yargs')
+  .usage('Usage: <file> [options]')
+
+  .alias('l', 'noLines')
+  .describe('l', 'skip line numbers in output to keep it more diff-friendly')
+
+  .alias('v', 'verbose')
+  .describe('v', 'also print relevant code')
+
+  .argv;
 
 import checkFile from './checker';
 
-const [,, tsFile] = process.argv;
+const [tsFile] = argv._;
+const { noLines, verbose } = argv;
 
-// TODO: read options from a tsconfig.json file.
-const options: ts.CompilerOptions = {};
-const host = ts.createCompilerHost(options, true);
+// read options from a tsconfig.json file.
+let host = ts.createCompilerHost({}, true);
+const options: ts.CompilerOptions = ts.readConfigFile('tsconfig.json', (path: string) => host.readFile(path)).config['compilerOptions'];
+// console.log(options);
+if (options) {
+  host = ts.createCompilerHost(options, true);
```

Comment:
This is fine for now, but eventually we should do something more [like tslint][1].

[1]: https://github.com/palantir/tslint/blob/595b1dbe332218d8c7ecadab9ee1a9f486233217/src/linter.ts#L73